ably/rest_client_test.go

```diff
@@ -195,3 +197,25 @@ var _ = Describe("RestClient", func() {
 		})
 	})
 })
+
+func TestRSC7(t *testing.T) {
+	app, err := ablytest.NewSandbox(nil)
+	if err != nil {
+		t.Fatal(err)
+	}
+	c, err := ably.NewRestClient(app.Options())
+	if err != nil {
+		t.Fatal(err)
+	}
+	t.Run("must set version header", func(ts *testing.T) {
+		req, err := c.NewHTTPRequest(&ably.Request{})
+		if err != nil {
+			ts.Fatal(err)
+		}
+		h := req.Header.Get("X-Ably-Version")
+		expect := "1.0"
```

Comment:
Should we use a constant so that this test doesn't break when we change the library version?

Reply:
There are constants, just not exported. I had a different thought on this, but I think your argument is stronger will update to use the constants.